asgiref/sync.py

```diff
@@ -137,7 +138,7 @@ def __init__(self, func):
             func = self.func
 
         future = loop.run_in_executor(
-            None,
+            self.executor,
```

Comment:
I don't see anywhere you set `self.executor`?

Reply:
In the class definition from line 112 above. 

(Presuming that it’s be used in the updated tests.)

Reply:
Right, right, that's class level, I thought it was in `__init__`.